edx_sga/static/js/src/edx_sga.js

```diff
@@ -34,16 +34,18 @@ function StaffGradedAssignmentXBlock(runtime, element) {
             var content = $(element).find('#sga-content').html(template(state));
 
             $(content).find('.finalize-upload').on('click', function() {
-              $.post(finalizeUploadUrl).success(
+              if (confirm('Are you sure you want to submit this file?')) {
+                $.post(finalizeUploadUrl).success(
                   function (state) {
-                      render(state);
-                  }
-              ).fail(
+                  render(state);
+                }
+                ).fail(
                   function () {
-                      state.error = gettext('Submission failed. Please contact your course instructor.');
-                      render(state);
+                    state.error = gettext('Submission failed. Please contact your course instructor.');
```

Comment:
Can't we place these strings in a string/constant file?
or its edX repo so we can't make these changes?

Reply:
that's a good suggestion. however, there are many other such strings on the file so I don't want to do a lot of changes. But yes we can definitely change because we have forked it.